src/core/helpers/localization.js

```diff
@@ -65,13 +65,14 @@ export const currencyToNumber = currency =>
  * Converts number to a local currency
  * @returns {string} Converted Number
  */
-const convertNumber = ({ currency, options, convertedPrice }) =>
+const convertNumber = ({ currency, options, convertedPrice }) => {
   new Intl.NumberFormat(undefined, {
```

Comment:
Does this work without a `return` statement?

Reply:
I see will fix this